src/ui/Toggle/utils.ts

```diff
@@ -0,0 +1,10 @@
+export function filterNumber(str: string): Array<number> {
+  if (typeof str === 'number') {
+    return [str];
+  } else if (typeof str !== 'string') {
+    return [];
+  }
```

Comment:
Since the param type is 'string', this case won't be exist I guess. 
```
filterNumber(111) 
// or
filterNumber({...})
```
Both will throw an type error from beginning.

Reply:
I think the error will be shown for TypeScript, but we are supporting pure JavaScript too.
anyway, I applied for the review, could you review it again? @AhyoungRyu

Reply:
Ahh I see. Thank you for the update :)